### ISSUE TITLE
Clang doesn’t like `*(int*)0=0;`

### ISSUE THREAD

**Issue by JanX2** [xiph/rnnoise#6]
The following code produces warnings in clang: `*(int*)0=0;`

> Indirection of non-volatile null pointer will be deleted, not trap

It recommends the following:
> Consider using __builtin_trap() or qualifying pointer with 'volatile'

**Comment by jmvalin** [comment]
Yeah, that's my lazy way to assert -- forgot to get rid of it. It indeed should not be in the code.

**Comment by wegylexy** [comment]
Perhaps add `-Wno-null-dereference` when compiling with clang to suppress the warning?

**Comment by emaste** [comment]
`*(int *)0=0` is gone as of https://github.com/xiph/rnnoise/commit/5c375a61487f13bad596068c481d24d985b3689a
